recruiter/forms.py

```diff
@@ -33,3 +34,8 @@ class UpdateRecruiterAccountSettingsForm(forms.ModelForm):
     class Meta:
         model = Recruiter
         fields = ["name", "company", "email", "phone_number"]
+
+    # Override the init method to add classes to the form fields
+    def __init__(self, *args, **kwargs):
+        super(UpdateRecruiterAccountSettingsForm, self).__init__(*args, **kwargs)
```

Comment:
You don't need to pass argument to `super()` in Python 3. Unless you need precise control over the superclass selection.

Reply:
Fixed.